js/controller.js

```diff
@@ -1643,12 +1643,16 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
           this.hideControlBar();
         }.bind(this)
       );
-      this.state.pluginsClickElement.click(
-        function() {
-          this.state.pluginsClickElement.removeClass('oo-showing');
-          this.mb.publish(OO.EVENTS.PLAY);
-        }.bind(this)
-      );
+
+      //PLAYER-4041: It seems like 'click' events started when a div's pointerEvents is initially 'none' but is changed to
+      //'auto' prior to the 'click' event ending will trigger 'click' event listeners on Android. We'll instead listen to
+      //'touchend' and 'touchcancel' on Android.
+      if (OO.isAndroid) {
+        this.state.pluginsClickElement.on('touchend touchcancel', this.resumePlaybackAfterClickthrough.bind(this));
```

Comment:
Should touchcancel handle this operation? May be user changed his mind and don't want to perform action?

Reply:
I think the touchcancel event implementation is browser specific:

https://developer.mozilla.org/en-US/docs/Web/Events/touchcancel

Including touchcancel was more of a fallback in case the end user triggers the event instead of touchend. If triggered, the video will just unpause.

Reply:
@knooyala I'd like to get this merged so QA can test it. Let me know what you think, thanks!

Reply:
@knooyala we're going to merge this to get QA started. Let me know if you have any concerns and we'll continue discussion